topo/node/juniper/juniper_test.go

```diff
@@ -256,12 +256,9 @@ func TestGRPCConfig(t *testing.T) {
 			want: []string{
 				"set system services extension-service request-response grpc ssl hot-reloading",
 				"set system services extension-service request-response grpc ssl use-pki",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config services GNMI",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config enable true",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config port 32767",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config transport-security true",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config certificate-id grpc-server-cert",
-				"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config listen-addresses 0.0.0.0",
+				"set system services extension-service request-response grpc ssl address 0.0.0.0",
```

Comment:
Where are these lines coming from? In juniper.go you've hardcoded the configuration to only apply the new nginx format but are expecting the old one here.